app/(admin)/openmailer/campaigns/[id]/page.tsx

```
@@ -79,6 +83,13 @@ export default async function OpenmailerCampaignDetailPage({
     : null;
   const activeSubscriberCount = activeSubscriberCountResult?.[0]?.count ?? 0;
 
+  const dailySentCount = await getDailySentCount();
+  const quotaLimit = getQuotaLimit();
+
+  const showSendPending =
```

ESLint

[error] '=' should be placed at the beginning of the line.

(style/operator-linebreak)